examples/mnist/train_mnist_model_parallel.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import argparse
```

Comment:
According to PEP8, 
```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.
```
https://www.python.org/dev/peps/pep-0008/#imports

So we need a blank line after `argparse`.